crates/env/Cargo.toml

```diff
@@ -1,58 +1,60 @@
 [package]
 name = "ink_env"
-version = "4.2.0"
-authors = ["Parity Technologies <admin@parity.io>", "Robin Freyler <robin@parity.io>"]
-edition = "2021"
-rust-version = "1.68"
 
-license = "Apache-2.0"
-readme = "README.md"
-repository = "https://github.com/paritytech/ink"
-documentation = "https://docs.rs/ink_env/"
-homepage = "https://www.parity.io/"
+edition.workspace = true
```

Comment:
Could you please keep the same ordering of the fields we had before? (in all `Cargo.toml`)

Reply:
the order: name | inherited properties | other properties made more sense to me... it is also more transparent and easier to maintain

Reply:
We can reorder them in another PR and automate ordering by CI

Reply:
done in-place

Reply:
It is still in a different order=)
<img width="1223" alt="image" src="https://github.com/paritytech/ink/assets/18346821/44039588-d5b3-4a37-a53c-98b7dab66f09">

Reply:
should be visible now